src/modules/tokens/services/token.filtering.service.ts

```diff
@@ -71,4 +75,26 @@ export class TokenFilteringService {
 
         return filteredTokens;
     }
+
+    async tokensByLiquidity(
```

Comment:
`tokensByLiquidityUSD`. We can also have token liquidity in actual tokens

Reply:
done